server.py

```diff
@@ -57,8 +70,46 @@ def serve_data():
             socketio.emit('sensor_data', reading)
             print('SEND TO CLIENT:', reading)
             print('\n')
+            sensor_log('update',reading)
         time.sleep(2)
 
+# Logging function for sensor data
+def sensor_log(cmd, data):
+    global pod_runs
+    global logging_sensor_data
+    global logged_sensors
+    global run_start_time
+
+    if cmd == 'start':
+        pod_runs += 1
+        f = open('static/data/run_' + str(pod_runs) + ".csv",'w')
+        f.write('sensor,time,value\n')
+        f.close()
+        run_start_time = datetime.now()
+        logging_sensor_data = True
+        socketio.emit('log_update', {'pod_runs':pod_runs,'sensors':logged_sensors})
+
+    elif cmd == 'update' and logging_sensor_data:
+        time_crt = datetime.strftime(datetime.now(), '%H:%M:%S')
+        time_passed = datetime.strftime(server_start_time + (datetime.now() - run_start_time), '%H:%M:%S')
+
+        f = open('static/data/run_' + str(pod_runs) + ".csv", 'a')
```

Comment:
This is going to get very expensive. I'll merge it for now because it works but we should think of a way to keep the files open for the duration of the logging.